src/objects/BatchedMesh.js

```diff
@@ -1002,6 +1020,109 @@ class BatchedMesh extends Mesh {
 
 	}
 
+	setInstanceCount( maxInstanceCount ) {
```

Comment:
Do you mind adding documentation for both new methods?

Reply:
done!